core/controllers/topics_and_skills_dashboard.py

```diff
@@ -167,6 +167,24 @@ def get(self):
         self.render_json(self.values)
 
 
+class SkillAssignmentsHandler(base.BaseHandler):
+    """Provides information about which topics contain the given skill."""
+
+    GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+
+    @acl_decorators.can_access_topics_and_skills_dashboard
+    def get(self, skill_id):
+        """Handles GET requests."""
+        assigned_topics = skill_services.get_all_topics_assigned_to_skill(
```

Comment:
They aren't topic objects, right? Maybe rename to `topic_assignments`? Not sure.
This makes it seem like the function is returning full topic objects.

Reply:
Done.